webpack.config.node.js

```diff
@@ -93,7 +93,7 @@ const webpackConfig = {
 	},
 	resolve: {
 		extensions: [ '', '.json', '.js', '.jsx' ],
-		root: [ path.join( __dirname, 'server' ), path.join( __dirname, 'client' ), __dirname ],
```

Comment:
Better put `__dirname` at the start of this array instead of at the end. That was my fix, changing the `entry` path didn't even cross my mind :D